main/src/cgeo/geocaching/ImageSelectActivity.java

```diff
@@ -344,6 +350,10 @@ private void showFailure() {
         showToast(getString(R.string.err_acquire_image_failed));
     }
 
+    private void showFailureUnsupportedImageFormat() {
```

Comment:
I don't think it's worth introducing a new method here (especially since the method is only used once).